Source/Fluxor/DependencyInjection/DependencyScanner.cs

```diff
@@ -112,5 +112,16 @@ internal static class DependencyScanner
 				return store;
 			});
 		}
+		 public static IEnumerable<Type> GetLoadedTypes(Assembly assembly)
```

Comment:
You have a mix of tabs and spaces in here, could you use only tabs please?

Reply:
Should be fixed now :-)